optuna/integration/lightgbm_tuner/__init__.py

```diff
@@ -11,9 +11,15 @@
 
 def train(*args, **kwargs):
     # type: (List[Any], Optional[Dict[Any, Any]]) -> Any
-    """Wrapper function of LightGBM API: train()
+    """Wrapper of LightGBM Training API to tune hyperparameters.
 
-    Arguments and keyword arguments for `lightgbm.train()` can be passed.
+    .. warning::
+
+        This feature is experimental. The interface can change in the future.
```

Comment:
It looks passive voice is appropriate.

```suggestion
        This feature is experimental. The interface can be changed in the future.
```

Reply:
[nits] "may be changed" might be more suitable.

Reply:
Thank you! I accepted your suggestion.

[TODO] I'll also fix the following comment:
https://github.com/pfnet/optuna/blob/b96a3960ba8a6b636c18a7999e9ab819e6051c83/optuna/integration/sklearn.py#L337
At that time, I'll replace `can` with `may`.